src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/DirectActionContext.kt

```diff
@@ -34,6 +34,7 @@ open class DirectActionContext(private val context: SparkWebContext,
     override fun queryString(): String? = request.queryString()
     override fun params(): Map<String, String> = request.params()
     override fun params(key: String): String = request.params(key)
+    override fun splat(): Array<String>? = request.splat()
```

Comment:
This allows routes with wildcards, like `documents/*` - we should probably switch over to using this for artefacts also, since we're currently doing something weird there with the paths in ignorance of this functionality.